src/packages/web/page.gleam

```diff
@@ -39,9 +39,11 @@ fn search_form(search_term: String) -> Node(t) {
     [attrs.class("search-form"), attrs.Attr("method", "GET")],
     [
       html.input([
+        attrs.id("search-input"),
```

Comment:
Let's use a data attribute rather than an id, it will make it more clear this is for JavaScript and not CSS.

Reply:
Targeting for the input is now being done with the `data-search-input` attribute.